pravega/zookeeper-operator
# Update alpine image to 3.15

Signed-off-by: Nishant Gupta <Nishant_Gupta3@dell.com>

### Change log description

Update the alpine image to 3.15

### Purpose of the change

Fixes #446

### What the code does

updates the alpine image from 3.14 to 3.15

### How to verify it

Install the zookeeper-operator with the update alpine image and the pod should be up and running
